SConstruct

```diff
@@ -1249,7 +1249,11 @@ libraries = {
 			"CPPDEFINES" : [
 				( "CCL_NAMESPACE_BEGIN", "namespace ccl {" ),
 				( "CCL_NAMESPACE_END", "}" ),
+				( "WITH_OCIO", "1"),
 				( "WITH_OSL", "1" ),
+				( "WITH_OPENSUBDIV", "1"),
+				( "WITH_OPENVDB", "1"),
+				( "WITH_NANOVDB", "1")
```

Comment:
Good catch.

This is one of the things that I'm least impressed by with Cycles - no real library expects you to know its original build options (and namespace even!) at the point of use. If Cycles were properly packaged as a standalone renderer then none of this would be necessary.

I know you're still working on some details of the VDB loading, but perhaps it's worth merging this one commit in isolation while you carry on with that?